cocos/editor-support/spine/SkeletonRenderer.cpp

```diff
@@ -98,7 +98,7 @@ SkeletonRenderer::~SkeletonRenderer () {
 	spSkeleton_dispose(_skeleton);
 	if (_atlas) spAtlas_dispose(_atlas);
 	if (_attachmentLoader) spAttachmentLoader_dispose(_attachmentLoader);
-	delete _worldVertices;
+	delete [] _worldVertices;
```

Comment:
Done. Here's the [pull request in the Spine runtime repository](https://github.com/EsotericSoftware/spine-runtimes/pull/702).

Thanks.